codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/ActionServiceSourceGenerator.scala

```diff
@@ -298,13 +298,13 @@ object ActionServiceSourceGenerator {
         |  extends ActionProvider[${service.className}] {
         |
         |  override final def serviceDescriptor: Descriptors.ServiceDescriptor =
-        |    ${service.fqn.parent.name}Proto.javaDescriptor.findServiceByName("${service.fqn.protoName}")
+        |    ${typeName(service.descriptorObject)}.javaDescriptor.findServiceByName("${service.fqn.protoName}")
         |
         |  override final def newHandler(context: ActionCreationContext): ${service.handlerName} =
         |    new ${service.handlerName}(actionFactory(context))
         |
         |  override final def additionalDescriptors: immutable.Seq[Descriptors.FileDescriptor] =
-        |    ${service.fqn.parent.name}Proto.javaDescriptor ::
+        |    ${typeName(service.descriptorObject)}.javaDescriptor ::
```

Comment:
I think I just fixed this and is merged already.

Reply:
you might be thinking of https://github.com/lightbend/akkaserverless-java-sdk/pull/475#discussion_r714766515 , which seems to be failing that PR and motivated me to do this cleanup :)

Reply:
yeah, thanks. I will rebase #475